inc/Telemetry/Telemetry.php

```diff
@@ -102,7 +102,7 @@ public function track_remote_data_blocks_usage( int $post_id, WP_Post $post ): v
 		}
 
 		// Regular expression to match all remote data blocks present in the post content.
-		$reg_exp = '/<!--\s{1}wp:remote-data-blocks\/([^\s]+)\s/';
+		$reg_exp = '/<!--\s{1}wp:remote-data-blocks\/([^\s]+)\s.*?-->/';
```

Comment:
This ensures that we capture the attributes set on the block. The downside of this is that, the giant remoteData attribute is also captured. I haven't been able to effectively use regex to have it skip that, so I've left it like this. Happy to get feedback on this, and see if there's another way to do it.

I had one restriction that I kept in mind - we should only make one pass through the matched blocks. This way, performance is always kept in mind.

Reply:
It's fine as long as we're not shipping the attributes

Reply:
Nope, we def aren't. They are ignored unless its the error block and even then it's just to get the mode value and not to ship it.